frontend/src/lib/components/Frontpage.svelte

```diff
@@ -3,72 +3,22 @@
 import { base } from "$app/paths";
 
 import CardDisplay from "$lib/components//DataDisplay/CardDisplay.svelte";
+import Card from "$lib/components/Card.svelte";
 import GalleryDisplay from "$lib/components/DataDisplay/GalleryDisplay.svelte";
 
 export let getStarted = "";
 
-export let items = [
-	{
-		header: "Was ist Mondey?",
-		summary:
-			"Mondey ist ein wissentschaftlich geprüftes Programm zure Dokumentation der Entwicklung von Kindern bis 6 Jahren.",
-		href: `${base}/info`,
-		button: "Mehr Info",
-	},
-	{
-		header: "Wozu ist Mondey gut?",
-		summary:
-			"Anhand ihrer Bewertungen der Fähigkeiten des Kindes erhalten sie Feedback zum Entwicklungsstand des Kindes und können so frühzeitig Fördermaßnahmen einleiten. Dies folgt einem übersichtlichen Ampelsystem.",
-		href: `${base}/info`,
-		button: "Mehr Info",
-	},
-	{
-		header: "Was umfasst Mondey?",
-		summary:
-			"Mondey umfasst unterschiedliche Entwicklungsbereiche wie von Kindern im Alter von 0 bis 6 Jahren. Dazu gehören unter anderem Grob-und feinmotorik, Wahrnehmung, Denkne, Sprache und Soziale Beziehungen.",
-		href: `${base}/info`,
-		button: "Mehr Info",
-	},
-	{
-		header: "Wie funktioniert Mondey?",
-		summary:
-			"Sie bewerten wie gut das Kind bestimmte Alltagshandlungen durchführen kann mit Hilfe einer Liste von Fragen.",
-		href: `${base}/info`,
-		button: "Mehr Info",
-	},
-	{
-		header: "Wo fange ich an?",
-		summary:
-			"Um zu beginnen, müssen sie sich registrieren und ein Profil für ihr Kind anlegen.",
-		href: getStarted,
-		button: "Los geht´s",
-	},
-	{
-		header: "dummy?",
-		summary:
-			"Lorem, ipsum dolor sit amet consectetur adipisicing elit. Impedit repellendus distinctio facilis! Voluptas corrupti recusandae sapiente doloribus voluptatem fugiat ducimus.",
-		href: `${base}/info`,
-		button: "Mehr Info",
-	},
-	{
-		header: "dummy?",
-		summary:
-			"Lorem, ipsum dolor sit amet consectetur adipisicing elit. Impedit repellendus distinctio facilis! Voluptas corrupti recusandae sapiente doloribus voluptatem fugiat ducimus.",
-		href: `${base}/info`,
-		button: "Mehr Info",
-	},
-	{
-		header: "dummy?",
-		summary:
-			"Lorem, ipsum dolor sit amet consectetur adipisicing elit. Impedit repellendus distinctio facilis! Voluptas corrupti recusandae sapiente doloribus voluptatem fugiat ducimus.",
-		href: `${base}/info`,
-		button: "Mehr Info",
-	},
-];
+export let items = [];
 
 const props = {};
 </script>
 
+<Card
```

Comment:
Since your Card component has hard-coded contents, `Card` might be a bit too generic of a name, maybe you could rename it to FrontPageCard.svelte or something similarly specific?

Reply:
Ok, I get it. Thank you!

Here I have a same question — can I rewrite this line quickly? @lkeegan

Reply:
You need to rename the Card.svelte file as well - I'd recommend doing this locally, making a commit & pushing. (It sounds like a lot but with some practice it can also be very quick :-))
It might also be possible to rename files from the github website these days, but I've never tried.

Reply:
Good, that'll help me get through this faster :)

@lkeegan and the last question! How often can I create new branches to accomplish tasks? Is it a problem for if I do it too often or for very small changes?

Reply:
You should create a new branch for every new feature, it's fine if it is only a small change, in fact small changes are good - it makes it much easier to review the Pull Request if it is small and self-contained.

Reply:
okey! Thank you for help and guidance!